::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 26 Jan 2018 00:28:41 +0100
Subject: [PATCH] linux.system.single: Dpkg::Use-Pty=0

Silence useless output, as suggested in [1].

[1] https://askubuntu.com/a/668859

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/single/init.yml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/linux/system/single/init.yml b/linux/system/single/init.yml
index c7c583b..5933451 100644
--- a/linux/system/single/init.yml
+++ b/linux/system/single/init.yml
@@ -25,6 +25,9 @@ parameters:
         config:
           compression-workaround:
             "Acquire::CompressionTypes::Order": "gz"
+        config:
+          silence-dpkg:
+            "Dpkg::Use-Pty": "0"
       kernel:
         modules:
           - nf_conntrack
